Javascript/app2.js

```diff
@@ -1,24 +1,13 @@
 'use strict';
 console.log('app2.js is now connected');
 
+function questions(){
```

Comment:
You'll want to wrap a function around each question so 

`function questionOne(){
    let myName = prompt('First is an easy one. Is my full first name Joseph?')
    console.log("this is the user input", myName)
    let lowerCaseName = myName.toLowerCase();
    console.log('this is answer lowercase', lowerCaseName)
    if(lowerCaseName === 'y' || lowerCaseName=== 'yes' ){
      //console.log('You are correct!');
      alert('You are correct!');
      userData++;
    }   else {
      //console.log('Sorry, that is wrong...')
      alert('Sorry, that is wrong...')
    };
    
    questionOne()`